.github/workflows/quick-start.yml

```diff
@@ -0,0 +1,47 @@
+name: quick-start
+
+on:
+  workflow_call:
+    inputs:
+      fixtures:
+        required: false
+        type: string
+      
+      superuser:
+        required: false
+        type: string
+
+      web_service:
+        required: false
+        type: string
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Start docker containers
+        run: docker compose up -d --build || ( docker compose logs >&2 && exit 1; )
+      - name: Wait for migrations to finish
+        run: |
+          echo "Waiting for migrations to complete..."
+          until ! docker compose exec -T web src/manage.py showmigrations | grep -q '\[ \]'; do
+            echo "Migrations not finished, waiting..."
+            sleep 3
+          done
+      - name: Load fixtures
+        if: ${{ inputs.fixtures != '' }}
+        run: docker compose exec -T web src/manage.py loaddata ${{ inputs.fixtures }}
+      - name: Create superuser
+        run: |
+          SUPERUSER="${{ inputs.superuser || 'admin' }}"
+          WEB_SERVICE="${{ inputs.web_service || 'web' }}"
```

Comment:
Since we're not testing logging in in this job, I don't think we have to make these variable, but as long as it's not required to pass them it's fine by me to leave this as is